change/@itwin-measure-tools-react-a682d053-c4ed-4aec-8a4c-4e33bbe76d16.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
since youre bumping the peers, this is technically a major version bump, however since this pkg is 0.x, we can get away with a minor bump

Reply:
im not 100% sure why we needed to update deps, is it because some of the replacement apis are only available in newer versions?

Reply:
a lot of the functions were promoted to public in versions >4.0, and AccuSnap.doSnapRequest was a new api created in the same pr that promoted a few of the apis
https://github.com/iTwin/itwinjs-core/pull/5720

Reply:
then youll need to update the changelog entry, and would be good to call out when the apis youre using were added in the description

Reply:
why is the type here set to `none` ?

Reply:
changing to minor